R/process_data_set_jordan.R

```diff
@@ -0,0 +1,33 @@
+library(tidyverse)
+library(usethis)
+library(stringr)
+library(lubridate)
+library(janitor)
+
+# Import dataset from csv
+fight_songs <- read_csv("data-raw/fight-songs/fight-songs.csv")
+
+names(fight_songs)[12] <- "num_fights"
+
+# Change variable types
+fight_songs <- fight_songs %>%
+  mutate(year = as.numeric(year),
```

Comment:
Could you move `year` to a new line, then re-indent all other variables (Can do this in RStudio by highlighting code and then pressing COMMAND+I on macOS, CONTROL + I on windows). Kind of like in the [tidyverse style guide](https://style.tidyverse.org/pipes.html#long-lines-2)